src/components/views/dialogs/SeshatResetDialog.js

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {_t} from "../../../languageHandler";
+import * as sdk from "../../../index";
+import {replaceableComponent} from "../../../utils/replaceableComponent";
+
+@replaceableComponent("views.dialogs.SeshatResetDialog")
+export default class SeshatResetDialog extends React.Component {
+    static propTypes = {
+        onFinished: PropTypes.func.isRequired,
+    };
+    render() {
+        const BaseDialog = sdk.getComponent('views.dialogs.BaseDialog');
+        const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
+
+        return (
+            <BaseDialog
+                    hasCancel={true}
+                    onFinished={this.props.onFinished.bind(null, false)}
+                    title={_t("Reset event index store?")}>
```

Comment:
minor lint: 4 space indent on attributes (and depending on who you ask, the `>` should be on its own line to wrap the attributes nicely)

Reply:
Interestingly enough as soon as I renamed my file from `.js` to `.tsx` ESLint picked the error in formatting
Cheers for the heads up!